tests/python/pytest_ops.py

```diff
@@ -190,8 +190,10 @@ def schedule(self):
 # TODO Maybe only test a single dtype
 @create_op_test(tuple(op for op in opinfos if op.sample_input_generator is not None))
 def test_definition_op_in_schedule_error(op: OpInfo, dtype: torch.dtype):
-    clear_cuda_cache()
     for sample in op.sample_input_generator(op, dtype):
+        # clear cache for each sample
+        FusionCache.reset()
+        clear_cuda_cache()
```

Comment:
should this one also be applied to `test_correctness` as well as `test_errors`

Reply:
We're already applying `clear_cuda_cache()` per-test.

Reply:
We accumulate memory because it is a user schedule error. In `finalize_schedule`, we release memory after scheduling, but it is skipped because of the error. Clearing the FusionCache deletes all `UserSchedules`.

Reply:
got ya. sorry I missed that int he PR description.

Do we not need it in `test_errors`? The test could still raise before `fianlize_schedule` in those tests and accumulating memory, but it might not have happened frequent enough there to raise any issue yet.

Reply:
If the schedule function is not defined in `FusionDefinition` sub-class, then `setup_schedule` is not run and the `KernelArgumentHolder` is not created.

Reply:
Got it. Thanks for walking me through this.

Reply:
I realized that you have it well written in the description. Apology for my incompetent reading skill. :face_holding_back_tears: 

> SchedulerRuntimeInfo is created in setupSchedule. Normally, finalizeSchedule is called after scheduling, which destroys runtime_info.